backend/schemas/dashboardTile.js

```diff
@@ -5,7 +5,11 @@ var tiles = mongoose.Schema({
     "displayTitle": String,
     "description": String,
     "_poster": String,
+    "_linkType": {type: String, default: 'course'},
+    "_courseLink": {type: mongoose.Schema.Types.ObjectId, ref:'Course'},
+    "_resourceUpload": String,
     "_link": String,
+    "_courseLink": String,
```

Comment:
Seems like two _courseLink's here...which one is it?

Reply:
That's a strange duplicate! It's the one on line 9.